script/DeployPlatform.s.sol

```diff
@@ -17,12 +17,12 @@ contract DeployPlatform is Script {
         address deployerAddress = vm.addr(deployerPrivateKey);
 
         // Goerli
-        //address platformColdWallet = 0x1695F52e342f3554eC8BC06621B7f5d1644cCE39;
-        //address platformAdminWallet = 0x1695F52e342f3554eC8BC06621B7f5d1644cCE39;
+        address platformColdWallet = 0x1695F52e342f3554eC8BC06621B7f5d1644cCE39;
+        address trustedForwarder = 0x0445d09A1917196E1DC12EdB7334C70c1FfB1623;
```

Comment:
why this new address?
this is very security critical

Reply:
This is our Goerli forwarder. The mainnet forwarder is 0xAa3E82b4c4093b4bA13Cb5714382C99ADBf750cA. Both is correct in the script.